scripts/install/opt/seagate/cortx/motr/conf/examples/motr_prov_conf.json

```diff
@@ -0,0 +1,78 @@
+{
+  "cluster": {
+    "server_nodes": {
+      "0a31807456be14965f31b49314f5664e": "srvnode_1",
```

Comment:
"0a31807456be14965f31b49314f5664e"

What is this?
Some kind of UUID?
How can I get this?